tokio/src/future/mod.rs

```diff
@@ -0,0 +1,15 @@
+#![allow(unused_imports, dead_code)]
```

Comment:
I don't really feel like figuring out which feature flags need which helpers. I hope that some of these can be removed.